api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/MotTestReadDaoJdbc.java

```diff
@@ -167,1895 +109,892 @@ public MotTest getMotTestByNumber(long number) {
 
-        logger.debug("Entry getMotTestCurrentsByVehicleId : " + vehicleId);
-        List<MotTest> motTests = new ArrayList<>();
-
-        logger.debug("Prepare getMotTestCurrentsByVehicleId : " + vehicleId);
-
-        try {
-            Connection connection = connectionManager.getConnection();
-
-            try (PreparedStatement stmt = connection.prepareStatement(new GetMotTestCurrentByVehicleId().buildQuery())) {
-                stmt.setInt(1, vehicleId);
-
-                logger.debug("Resultset getMotTestCurrentsByVehicleId : " + vehicleId);
-                try (ResultSet resultSet = stmt.executeQuery()) {
-                    while (resultSet.next()) {
-                        logger.debug("Map getMotTestCurrentsByVehicleId : " + vehicleId);
-                        MotTest motTest = mapResultSetToMotTestCurrent(resultSet);
-                        motTests.add(motTest);
-
-                    }
-                }
-            }
-        } catch (SQLException e) {
-            logger.error("Map getMotTestCurrentsByVehicleId : " + vehicleId, e);
-            throw new InternalException(e);
-        }
-
-        return motTests;
+        DbQueryRunner runner = new DbQueryRunnerImpl(connectionManager.getConnection());
+        ResultSetMapper<MotTest> mapper = new MotTestCurrentMapper();
+        return runner.executeQueryForList(new GetMotTestCurrentByVehicleId().buildQuery(), mapper, vehicleId);
     }
 
     @Override
     public List<MotTest> getMotTestHistorysByVehicleId(int vehicleId) {
 
-        logger.debug("Entry getMotTestHistorysByVehicleId : " + vehicleId);
-        List<MotTest> motTests = new ArrayList<>();
-
-        logger.debug("Prepare getMotTestHistorysByVehicleId : " + vehicleId);
-
-        try {
-            Connection connection = connectionManager.getConnection();
-
-            try (PreparedStatement stmt = connection.prepareStatement(new GetMotTestHistoryByVehicleId().buildQuery())) {
-                stmt.setInt(1, vehicleId);
-
-                logger.debug("Resultset getMotTestHistorysByVehicleId : " + vehicleId);
-                try (ResultSet resultSet = stmt.executeQuery()) {
-                    while (resultSet.next()) {
-                        logger.debug("Map getMotTestHistorysByVehicleId : " + vehicleId);
-                        MotTest motTest = mapResultSetToMotTestHistory(resultSet);
-                        motTests.add(motTest);
-                    }
-                }
-            }
-        } catch (SQLException e) {
-            logger.error("Error getMotTestHistorysByVehicleId : " + vehicleId, e);
-            throw new InternalException(e);
-        }
-
-        return motTests;
+        DbQueryRunner runner = new DbQueryRunnerImpl(connectionManager.getConnection());
+        ResultSetMapper<MotTest> mapper = new MotTestHistoryMapper();
+        return runner.executeQueryForList(new GetMotTestHistoryByVehicleId().buildQuery(), mapper, vehicleId);
```

Comment:
I think convention is separating the return statement from the rest of the body with a single space in our projects.